### PR TITLE
Add the kourier repository and create the basic image.

### PR DESCRIPTION
**Pull request by markusthoemmes**
We need to build and productize the kourier images as part of our productization pipeline.

### CODE REVIEW

File: ci-operator/config/openshift-knative/kourier/openshift-knative-kourier-master.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- from: base
```
Comment (mgencur) [inline ci-operator/config/openshift-knative/kourier/openshift-knative-kourier-master.yaml]: Is something missing here? Or the context_dir will be the root directory where the Dockerfile is located?

Comment (markusthoemmes) [reply ci-operator/config/openshift-knative/kourier/openshift-knative-kourier-master.yaml]: The latter, this is the same in the serving-operator

**Comment by mgencur** [comment]
I don't have permissions to merge in this folder yet. Please review.

/assign @alvaroaleman @sdodson

**Comment by sdodson** [comment]
/approve
/hold
@petr-muller clear if you're ok, didn't know if you had concerns since you were triggering rehearsals

**Comment by sdodson** [comment]
Ahh, my approval wasn't complete, Petr can complete it.
/hold cancel

**Pull request merged**